app/views/posts/show.html.slim

```diff
@@ -1,12 +1,16 @@
 .dflex.justify-content-center
   .card style="padding:2% 5% 2% 5%"
     small.text-muted
-      | Posted by&#160
+      | Posted by
       i
         = User.find(@post.user_id).username
-      | &#160&#183
     h4.card-title
       = @post.title
+      - if user_signed_in?
+        - if BookmarkedPost.find_by(user_id: current_user.id, post_id: @post.id) == nil
```

Comment:
Šeit būtu labak ja definetu instance mainigo iekš kontroleri:
 @bookmarked = BookmarkedPost.find_by(user_id: current_user.id, post_id: @post.id).blank?
Un tad iekš view tu varētu izmantot ternary condition (Palasi par to!)

```
- if user_signed_in?
  i#bookmark_icon.bi class="@bookmarked ? 'bi-bookmark' : 'bi-bookmark-fill'" data-id="#{@post.id}"
```

Reply:
Centies ari izmantot Slim specific klases un Id padošanu:

`span#button.button-big`